puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/builders/gemset_create.yaml

```diff
@@ -0,0 +1,4 @@
+- builder:
+    name: gemset_create
+    builders:
+      - shell: !include-raw scripts/gemset_create.sh
```

Comment:
It could be the shebang line in the scripts creating a new shell, the plugin may rely on using the default shell (I've not checked).